components/breadcrumb/src/Crumb.js

```diff
@@ -2,102 +2,61 @@
 
 import { jsx, useBrand, overrideReconciler } from '@westpac/core';
 import PropTypes from 'prop-types';
-import React from 'react';
 
-import { Crumb as CrumbWrapper, crumbStyles } from './overrides/crumb';
-import { Link, linkStyles } from './overrides/link';
-import { Icon, iconStyles } from './overrides/icon';
+import { useBreadcrumbContext } from './Breadcrumb';
```

Comment:
Can we keep the overrides visually separate from the dependencies (in this case the context)?